OSDOCS-3596: Release note for MS HyperVGeneration version 2 support (#45295)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

[id="ocp-4-11-installation-and-upgrade"]
=== Installation and upgrade

[id="ocp-4-11-installation-and-upgrade-hyperv"]
==== Support for Microsoft Hyper-V generation version 2

> By default, the installation program now deploys a Microsoft Azure cluster using Hyper-V generation version 2 virtual machines (VMs). If the installation program detects that the instance type selected for the VMs does not support version 2, it falls back to version 1 for the deployment.

Reviewer: "reverts" instead of "falls back"? Not sure what is standard here. Your call.

Author: I think fall backs works here. IMO "reverts" implies to undo.

> By default, the installation program now deploys a Microsoft Azure and a Microsoft Azure Stack Hub cluster using a Hyper-V generation 2 image. If the installation program detects that the virtual machine cannot support a generation 2 image, it falls back to a generation 1 image for the deployment.

Author: Agreed. Thanks for the suggestion.

> By default, the installation program now deploys a Microsoft Azure and a Microsoft Azure Stack Hub cluster using a Hyper-V generation version 2 image. If the installation program detects that the virtual machine cannot support a version 2 image, it falls back to a version 1 image for the deployment.

Reviewer 2: I overlooked this but we need to make this a bit more clear. We now support/default virtual machines on Azure with Hyper-V generation version 2. It's more of a VM disk configuration feature rather than an image feature. Terraform controls this by the images deployed to the VMs so they're correlated but let's make it clear it's a VM feature.

```suggestion
By default, the installation program now deploys a Microsoft Azure and a Microsoft Azure Stack Hub cluster using Hyper-V Generation V2 VMs. If the installation program detects that the instance type selected for the virtual machines do not support version 2, it falls back to a version 1 for the deployment.
```

Author: Thank you for the clarifications.

Reviewer 3: 1. most VM HyperVGenerations ==['V1, V2]', previous we use V1 image, now we use V2 image on the osDisk of the VM instance. 
2. previous version we can not use the VM HyperVGenerations='V2', now we can use it. such as the vm in the standardDCSv3Family, standardDDCSv3Family
3. How about changed to "cluster using Hyper-V Generation V2 images" ?

> By default, the installation program now deploys a Microsoft Azure and a Microsoft Azure Stack Hub cluster using Hyper-V generation version 2 virtual machines (VMs). If the installation program detects that the instance type selected for the VMs do not support version 2, it falls back to version 1 for the deployment.

Reviewer 4: ```suggestion
By default, the installation program now deploys a Microsoft Azure  cluster using Hyper-V generation version 2 virtual machines (VMs). If the installation program detects that the instance type selected for the VMs does not support version 2, it falls back to version 1 for the deployment.
```

Sorry @mjpytlak, I misled you earlier in my Jira comment. We only made this change in Azure--not Azure Stack Hub.
